docs/resources/federated_settings_org_role_mapping.md

```diff
@@ -30,9 +30,9 @@ resource "mongodbatlas_federated_settings_org_role_mapping" "org_group_role_mapp
 ## Argument Reference
 
 * `federation_settings_id` - (Required) Unique 24-hexadecimal digit string that identifies the federated authentication configuration.
-* `org_id` - Unique 24-hexadecimal digit string that identifies the organization that contains your projects.
-* `external_group_name` - Unique human-readable label that identifies the identity provider group to which this role mapping applies.
-* `role_assignments` - Atlas roles and the unique identifiers of the groups and organizations associated with each role.
+* `org_id` - (Required) Unique 24-hexadecimal digit string that identifies the organization that contains your projects.
+* `external_group_name` - (Required) Unique human-readable label that identifies the identity provider group to which this role mapping applies.
```

Comment:
```suggestion
* `external_group_name` - (Required) Unique label that identifies the identity provider group to which this role mapping applies.
```
We want to avoid 'human-readable', it doesn't add any clarity.